security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,45 @@ These release notes track the development of {cert-manager-operator}.
+
+Issued: 2025-11-
+
+The following advisories are available for the {cert-manager-operator} 1.18.0:
+
+* link:https://access.redhat.com/errata/RH[RH]
+
+Version `1.18.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.18.3`. For more information, see the link:https://https://cert-manager.io/docs/releases/release-notes/release-notes-1.18#v1183[cert-manager project release notes for v1.18.3].
+
+[id="cert-manager-operator-1-18-0-features-enhancements_{context}"]
+=== New features and enhancements
+
+*Replica count configuration for {cert-manager-operator} operands*
+
+With this release, you can override the default replica counts for the {cert-manager-operator} `controller`, `webhook`, and `cainjector` operands. To configure these values, specify the new `overrideReplicas` fields in the CertManager custom resource. With this enhancement, you can configure high availability (HA) and scale operands based on your specific operational requirements. For more information, see ---.
+
+*Root filesystem is read-only for {cert-manager-operator} containers*
+
+With this release, to improve security, the {cert-manager-operator} and all its operands have the `readOnlyRootFilesystem` security context set to `true` by default. This enhancement hardens the containers and prevents a potential attacker from modifying the contents of the container's root file system.
+
+*Istio-CSR integration with {cert-manager-operator} (Generally Available)*
+
+With this release, the integration of the {cert-manager-operator} with Istio-CSR, which was previously provided as a Technology Preview feature, is fully supported. This feature offers enhanced support for securing workloads and control plane components within Red Hat OpenShift Service Mesh or Istio environments. By utilizing the {cert-manager-operator} managed Istio-CSR agent, Istio can obtain, sign, deliver, and renew certificates required for mutual TLS (mTLS).
+For more information, see xref:../../security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc#cert-manager-operator-istio-csr-installing_cert-manager-operator-integrating-istio[Integrating the cert-manager Operator with Istio-CSR]. 
+
+*Network policy hardening is now available for cert-manager components*
+
+With this release, {cert-manager-operator} includes pre-defined `NetworkPolicy` resources designed for enhanced security by governing ingress and egress traffic for operand components. These policies cover essential internal traffic, such as ingress to the metrics and webhook servers, and egress to the OpenShift API server and DNS server. Note that deployment of the `NetworkPolicy` is not enabled by default to minimize impact during upgrades, so it must be explicitly enabled within the `CertManager` custom resource. Users who require connectivity to external issuer providers must configure additional egress allowance policies using the `CertManager` custom resource. For more information, see ---.
```

Comment:
```suggestion
With this release, {cert-manager-operator} includes pre-defined `NetworkPolicy` resources designed for enhanced security by governing ingress and egress traffic for operand components. These policies cover essential internal traffic, such as ingress to the metrics and webhook servers, and egress to the OpenShift API server and DNS server. Note that deployment of the `NetworkPolicy` is not enabled by default to minimize impact during upgrades, so it must be explicitly enabled within the `CertManager` custom resource. For more information, see ---.
```

To avoid redundant contents